### PR TITLE
[#87] feature/87 Load test dss rest server performance

### LINKED ISSUE #87: Load test dss rest server performance
Make a simple(echo) rest server, deploy to a cloud server and run it.
Use gatling(https://gatling.io/) or other tool to test server performance.

And push the echo server code, jar(jave archive) and test results under benchmark repository.

### PR DESCRIPTION
**Pull request by ParkSale**
Resolves #87

I use gatling for test server performance

I use simple echo server for refering to the tutorial and test it

Please review simple echo server and another something wrong

### CODE REVIEW

File: benchmark/dss/20200812/src/main/java/Application.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import io.github.ztkmkoo.dss.server.rest.DssRestServer;
```
Comment (ztkmkoo) [inline benchmark/dss/20200812/src/main/java/Application.java:1]: You'd better declare how to import this. (I think i cannot build this files)
It is better to add a maven or gralde config file.

Comment (ParkSale) [reply benchmark/dss/20200812/src/main/java/Application.java:1]: ah, I forgot this. Thank you.

File: benchmark/dss/20200812/src/main/java/MyServiceResponse.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import io.github.ztkmkoo.dss.core.actor.rest.entity.DssRestServiceResponse;
+
+public class MyServiceResponse implements DssRestServiceResponse {
```
Comment (ztkmkoo) [inline benchmark/dss/20200812/src/main/java/MyServiceResponse.java:3]: DssRestServiceResponse is serializable.
You should add serialVersionUID

Comment (ztkmkoo) [reply benchmark/dss/20200812/src/main/java/MyServiceResponse.java:3]: ref link: https://woowabros.github.io/experience/2017/10/17/java-serialize2.html

File: benchmark/dss/20200812/RecordedSimulation.scala [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+		nothingFor(4 seconds),
+		atOnceUsers(10),
+		rampUsers(10) during (5 seconds),
+		constantUsersPerSec(20) during (15 seconds),
+		constantUsersPerSec(20) during (15 seconds) randomized,
+		rampUsersPerSec(10) to 20 during (10 minutes),
+		rampUsersPerSec(10) to 20 during (10 minutes) randomized,
+		heavisideUsers(1000) during (20 seconds)
+	).protocols(httpProtocol)
+)
```
Comment (Alencion) [inline benchmark/dss/20200812/RecordedSimulation.scala:38]: I think it's awkward to indent this part

Comment (ParkSale) [reply benchmark/dss/20200812/RecordedSimulation.scala:38]: thank you for review. I'll fix it right now

**Comment by ParkSale** [comment]
It's my first time to rebase. So, there are so many commits.. 😭

**Pull request merged**